integration-tests/elasticsearch-rest-client/src/main/java/io/quarkus/it/elasticsearch/FruitService.java

```diff
@@ -0,0 +1,68 @@
+package io.quarkus.it.elasticsearch;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.apache.http.util.EntityUtils;
+import org.elasticsearch.client.Request;
+import org.elasticsearch.client.Response;
+import org.elasticsearch.client.RestClient;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+@ApplicationScoped
+public class FruitService {
+    @Inject
+    RestClient restClient;
+
+    public void index(Fruit fruit) throws IOException {
+        Request request = new Request(
+                "PUT",
+                "/fruits/_doc/" + fruit.id);
+        request.setJsonEntity(JsonObject.mapFrom(fruit).toString());
+        restClient.performRequest(request);
```

Comment:
Since we're all about reactive these days, maybe you could use `performRequestAsync` instead? It's not exactly reactive, but it's async and accepts a callback, so...

The code would definitely be more complex, though.

Reply:
Yeah, let's keep it simple in the tests IMHO.